src/crud/document.py

```
@@ -43,11 +63,11 @@
     )
     if max_distance is not None:
         stmt = stmt.where(
-            models.Document.embedding.cosine_distance(embedding_query) < max_distance
+            models.Document.embedding.cosine_distance(embedding) < max_distance
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)